pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -317,6 +317,9 @@
     public static final String ACCESS_CONTROL_FACTORY_CLASS = "pinot.server.admin.access.control.factory.class";
     public static final String DEFAULT_ACCESS_CONTROL_FACTORY_CLASS =
         "org.apache.pinot.server.api.access.AllowAllAccessFactory";
+
+    public static final String CONFIG_OF_ENABLE_THREAD_CPU_TIME_INSTRUMENT = "pinot.server.instance.enableThreadCpuTimeInstrument";
```

Comment:
(nit) rename this to pinot.server.instance.enableThreadCpuTimeMeasurement

Reply:
done